CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.36]
+### Fixed
+- Empty input sentence handling during inference
```

Comment:
Could you provide a bit more detail, e.g. "Fixed issue with the order of translations when empty inputs are present."

Reply:
Done.